pkg/plugins/builder.go

```diff
@@ -110,37 +110,23 @@ func NewBuilder(client *Client, plugins map[string]Descriptor, localPlugins map[
 	}
 
 	return pb, nil
-}
 
-func initMiddlewareBuilder(logger zerolog.Logger, goPath string, moduleName string, manifest *Manifest) (*interp.Interpreter, string, error) {
-	if !manifest.IsYaegiPlugin() {
-		return nil, filepath.Join(goPath, "src", moduleName, manifest.WasmPath), nil
-	}
-	i, err := initInterp(logger, goPath, manifest.Import)
-	return i, "", err
 }
 
-func initInterp(logger zerolog.Logger, goPath string, manifestImport string) (*interp.Interpreter, error) {
-	i := interp.New(interp.Options{
-		GoPath: goPath,
-		Env:    os.Environ(),
-		Stdout: logs.NoLevel(logger, zerolog.DebugLevel),
-		Stderr: logs.NoLevel(logger, zerolog.ErrorLevel),
-	})
-
-	err := i.Use(stdlib.Symbols)
-	if err != nil {
-		return nil, fmt.Errorf("failed to load symbols: %w", err)
+// Build builds a middleware plugin.
+func (b Builder) Build(pName string, config map[string]interface{}, middlewareName string) (Constructor, error) {
+	if b.middlewareBuilders == nil {
+		return nil, fmt.Errorf("no plugin definitions in the static configuration: %s", pName)
 	}
 
-	err = i.Use(ppSymbols())
-	if err != nil {
-		return nil, fmt.Errorf("failed to load provider symbols: %w", err)
+	// plugin (pName) can be located in yaegi middleware builders.
```

Comment:
middlewareBuilders can contain now yaegi and wasm middlewares. Perhaps worth of either removing yaegi from the comment or adding both yaegi and wasm there